frontend/public/components/cluster-settings/cluster-version.tsx

```diff
@@ -18,7 +18,7 @@ const ClusterVersionDetails: React.FC<ClusterVersionDetailsProps> = ({ obj }) =>
     <>
       <div className="co-m-pane__body">
         <SectionHeading text={t('public~ClusterVersion details')} />
-        <ResourceSummary resource={obj} />
+        <ResourceSummary resource={obj} showUpstreamConfig />
```

Comment:
We don't want to add `showUpstreamConfig` to `ResourceSummary` since it's very specific to the `ClusterVersion` resource and `ResourceSummary` is meant to be a general component. You should be able to pass it as a child to `ResourceSummary`, though.

```suggestion
        <ResourceSummary resource={obj}>
          <UpstreamConfigDetailsItem resource={obj} />
        </ResourceSummary>
```

Reply:
Done! Tested and works locally: 
![image](https://user-images.githubusercontent.com/21317056/131898962-442dd1a8-7e57-4882-bd08-db4d7142f831.png)